src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,64 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    public static final int ZERO = 0;
+    public static final String DEFAULT_DELIMITER = ",|:";
+    public static final String CUSTOM_DELIMITER_FIND_REGEX = "//(.*)\n(.*)";
+
+    public static int splitAndSum(String text) {
+        int sum = ZERO;
+        if (isBlank(text)) {
+            return sum;
+        }
+
+        sum = sum(getNumbers(split(text)));
```

Comment:
```suggestion
        sum = sum(toNumbers(split(text)));
```
- get prefix는 객체의 필드를 꺼내는 정도나, 값을 꺼내는 과정에서 약간의 변경을 위해 override하는 정도로만 활용하는건 어떨까요? 다른 기능을 하는 경우 더 적절한 네이밍을 하는 것이 좋다고 생각해요 (안그러면 대부분의 메소드이름이 get, set이 되어버려서요;)

Reply:
메소드이름 정하는게 제일 어려운거같아요 ㅜㅠ 
혹시 노하우나 참고할만한 글이있을까요??

Reply:
ㅎㅎ; 네이밍은 영어 문제라 뭐라 추천드릴만한게 없네요.
gpt에 물어보는건 어떨까요?